src/main/java/adris/altoclef/trackers/InventoryTracker.java

```diff
@@ -123,6 +123,30 @@ public boolean hasItem(String ...catalogueNames) {
         return false;
     }
 
+    public boolean swapToWeapon() {
```

Comment:
Sorry if this is nitpicky, but perhaps a better description? Swap to "what" weapon?

Reply:
added javadoc for it, explaining what it does and also renamed the method to "swapToSword"